### PR TITLE
Erased not needed files in models and changed the model

### PR DESCRIPTION
**Pull request by WiktorSa**
Only kept files that are needed in the model section.

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (sgalella) [inline .idea/.gitignore:1]: Can you add `.idea` to the `.gitignore`? These are settings for your IDE.

File: models/model.py [hunk 2]
```diff
@@ -1,24 +1,47 @@
...
-            nn.BatchNorm2d(num_features=40),
-            nn.ELU(),  # TODO: Add dropout
-            nn.Conv2d(in_channels=40, out_channels=40, kernel_size=(62, 1)),  # Spatial Conv.
-            nn.BatchNorm2d(num_features=40),
-            nn.ELU(),
-            nn.AvgPool2d(kernel_size=(1, 56), stride=(1, 14)),
-            nn.Flatten(start_dim=1),
-            nn.Linear(1200, 4)
-        )
+        dilation = 1
```
Comment (sgalella) [inline models/model.py:34]: Remove dilation variable (the default is 1).

File: models/model.py [hunk 3]
```diff
@@ -1,24 +1,47 @@
...
+        """
+        Temporal convolution block based on a residual block from Sulabh Kumra and Christopher Kanan 2017
+        https://www.researchgate.net/publication/310953119_Robotic_Grasp_Detection_using_Deep_Convolutional_Neural_Networks
+        """
+        super().__init__()
+        # 1. Padding on the left side
+        pad = torch.nn.ZeroPad2d((padding, 0, 0, 0))
+        # 2. Convolutional network (we didnt include weight norms)
+        conv2d1 = nn.Conv2d(in_channels=in_channels, out_channels=out_channels, kernel_size=(1, kernel_size),
+                            stride=stride, padding=0, dilation=dilation)
```
Comment (sgalella) [inline models/model.py:15]: Default value of padding is 0.

**Review by sgalella — CHANGES_REQUESTED** [review]
Can you add the model without deleting the one existing? Also, I still use the other files, can you leave them there? Once we have organized the structure of the files and integrated the preprocessing part we can delete them without any problem.

File: models/model.py (hunk 2 above)
Comment (WiktorSa) [reply models/model.py]: Fixed

File: models/model.py (hunk 3 above)
Comment (WiktorSa) [reply models/model.py]: Fixed

**Comment by WiktorSa** [comment]
I fixed all the things. Nothing should crash.

**Review by sgalella — APPROVED** [review]
